pkg/apis/compliance/v1alpha1/compliancecheckresult_types.go

```diff
@@ -14,6 +14,11 @@ type ComplianceCheckStatus string
 const ComplianceCheckResultStatusLabel = "compliance.openshift.io/check-status"
 const ComplianceCheckResultSeverityLabel = "compliance.openshift.io/check-severity"
 
+// ComplianceCheckResultLabel defines a label that will be included in the
+// ComplianceCheckResult objects. It indicates whether the result has an automated
+// remediation or not.
+const ComplianceCheckResultHasRemediation = "compliance.openshift.io/automated-remediation"
```

Comment:
Can you add docs about this? Also about its usage;

e.g.

* How do I search for results that have automated remediations?
* How do I search for results without automated remediations?

Reply:
Done